iosApp/PhongKMMIC/Sources/Presentation/Modules/Home/Views/SurveyContentView.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  SurveyContentVIew.swift
+//  PhongKMMIC
+//
+//  Created by Phong Vo on 07/05/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct SurveyContentView: View {
+
+    var uiModels: [UIModel]
+    @State private var currentPage = 0
+    @State private var nextPage = 0
+    @State private var currentVisibility: Double = 1.0
+
+    @GestureVelocity private var velocity: CGVector
+
+    private var draggingSpeed: Double = 700.0
+    private var draggingLength: Double = 160.0
+    private var draggingVisibilityMultiplier: Double = 1.2
+
+    // swiftlint:disable closure_body_length
+    var body: some View {
+        GeometryReader { geometryReader in
+            VStack {
+                PageViewIndicator(numberOfPage: uiModels.count, currentPage: $currentPage)
+                    .frame(width: geometryReader.size.width - 40.0, alignment: .leading)
+                    .padding([.bottom], 20.0)
+                Text(uiModels[safe: currentPage]?.title ?? "")
+                    .foregroundColor(.white)
+                    .font(.boldTitle)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                    .multilineTextAlignment(.leading)
+                    .lineLimit(2)
+                HStack {
+                    Text(uiModels[safe: currentPage]?.description ?? "")
+                        .foregroundColor(.white.opacity(0.7))
+                        .font(.regularBody)
+                        .lineLimit(2)
+                    Spacer()
+                    Button {
+                        // TODO: Handle button taps
+                        print("Next button was tapped")
+                    } label: {
+                        R.image.rightChevron.image
+                            .frame(width: 56.0, height: 56.0)
+                            .background(Color.white)
+                            .clipShape(Circle())
+                    }
+                }
+            }
+            .padding([.horizontal], 20.0)
+            .frame(
+                width: geometryReader.size.width,
+                height: geometryReader.size.height,
+                alignment: .bottom
+            )
+            .background(content: {
+                if let image = uiModels[safe: currentPage]?.image {
```

Comment:
What's happened when the image is from an URL? 🤔

Reply:
I've already left an extension of `KingFisher` for that. That will be done in the integrated story. However, I'll update it with an actually url

Reply:
solved in 149e645